www/apps/docs/content/deployments/server/deploying-on-railway.md

```diff
@@ -373,11 +357,11 @@ This error may be thrown by a module that uses Redis. If you see it in your buil
 
 ---
 
-## Run Commands on the Backend
+## Run Commands locally with the service variables available
```

Comment:
I don't think this title makes things clearer. Maybe:

```suggestion
## Run Commands Locally on the Deployed Backend
```

And then in the following text we mention that to run commands on your deployed backend, you run the command locally using Railway's CLI tool

Reply:
Then I must've misunderstood the use of the Railway CLI. The point of this section is to point users to how they can run command on their deployed backend, such as to create a user. What in your opinion is the alternative here?

Reply:
The help text for `railway run` is as follows "Run a local command using variables from the active environment"

So to say "Run Commands on the Deployed Backend" is misleading, but I'm not saying it won't work, just that the wording needs to change.